# This contains a list of Odometry type messages
std_msgs/Header header
nav_msgs/Odometry[] array